src/AppStore.ts

```diff
@@ -71,6 +63,18 @@ export class AppStore {
         this._appReady = true;
     }
 
+    @autobind
+    handleServiceError(error: SiteError): void {
+        if (
+            error.errorObj.status &&
+            parseInt(error.errorObj.status) &&
+            parseInt(error.errorObj.status) >= 400
```

Comment:
this probably needs refinement

Reply:
I'm not sure I understand this condition
```
parseInt(error.errorObj.status) &&
parseInt(error.errorObj.status) >= 400
```

would it not be `error.errorObj.status && parseInt(..)>=400`? otherwise `parseInt(.) && parseInt(..)>=400` is redundant?

Reply:
i think i didn't want to do comparison on what will be NaN (parseInt(undefind)).  It turns out to work.

Reply:
it's a little hard to read. can you do `!isNaN(error.errorObj.status)`?